.travis/test_06_script.sh

```diff
@@ -41,12 +41,19 @@ DOCKER_EXEC ./configure --cache-file=../config.cache $BITCOIN_CONFIG_ALL $BITCOI
 END_FOLD
 
 BEGIN_FOLD build
-DOCKER_EXEC make $MAKEJOBS $GOAL || ( echo "Build failure. Verbose build follows." && DOCKER_EXEC make $GOAL V=1 ; false )
+DOCKER_EXEC make $MAKEJOBS $GOAL || (
+  echo "Build failure. Verbose build follows." && DOCKER_EXEC make $GOAL V=1
+  DOCKER_EXEC "cat ${TRAVIS_BUILD_DIR}/sanitizer-output/* 2> /dev/null"
+  false
+)
```

Comment:
Could these commands be run in `after_failure` step?

Reply:
@ken2812221 I haven't tried `after_failure` before, but let me try to summarise: the suggestion is to add script say `.travis/test_XX_after_failure.sh` containing ...

```
#!/bin/bash

DOCKER_EXEC "cat ${TRAVIS_BUILD_DIR}/sanitizer-output/* 2> /dev/null"
```

.. that would be referenced using ...

```
after_failure:
  - set -o errexit; source .travis/test_XX_after_failure.sh
```

… in `.travis.yml`?

Is that a correct summary of the suggestion? :-)

Reply:
@ken2812221 Friendly ping :-)

Reply:
Could just test it?

Reply:
Ah, I guess it wouldn't work because we exit "hard" as opposed to just return a non-zero exit code.

Reply:
The trap way seems to work. 8d78bb9536e44e6b79532f539b1dd0066ce7ee6c

 https://travis-ci.org/ken2812221/bitcoin/jobs/468068901